lectures/dynamic_programming/coleman_policy_iter.md

```diff
@@ -463,21 +464,24 @@ solution.
 Here's an object containing data from the log-linear growth model we used in the {doc}`value function iteration lecture <../dynamic_programming/optgrowth>`
 
 ```{code-cell} julia
-isoelastic(c, γ) = isone(γ) ? log(c) : (c^(1 - γ) - 1) / (1 - γ)
-Model = @with_kw (α = 0.65,                            # Productivity parameter
-                  β = 0.95,                            # Discount factor
-                  γ = 1.0,                             # Risk aversion
-                  μ = 0.0,                             # First parameter in lognorm(μ, σ)
-                  s = 0.1,                             # Second parameter in lognorm(μ, σ)
-                  grid = range(1e-6, 4, length = 200), # Grid
-                  grid_min = 1e-6,                     # Smallest grid point
-                  grid_max = 4.0,                      # Largest grid point
-                  grid_size = 200,                     # Number of grid points
-                  u = (c, γ = γ) -> isoelastic(c, γ),  # utility function
-                  ∂u∂c = c -> c^(-γ),                  # u′
-                  f = k -> k^α,                        # production function
-                  f′ = k -> α * k^(α - 1),             # f′
-                  )
+isoelastic(c, gamma) = isone(gamma) ? log(c) : (c^(1 - gamma) - 1) / (1 - gamma)
+function Model(; alpha = 0.65,                            # Productivity parameter
+               beta = 0.95,                            # Discount factor
+               gamma = 1.0,                             # Risk aversion
+               mu = 0.0,                             # First parameter in lognorm(mu, sigma)
+               s = 0.1,                             # Second parameter in lognorm(mu, sigma)
+               grid = range(1e-6, 4, length = 200), # Grid
+               grid_min = 1e-6,                     # Smallest grid point
+               grid_max = 4.0,                      # Largest grid point
+               grid_size = 200,                     # Number of grid points
+               u = (c, gamma = gamma) -> isoelastic(c, gamma),  # utility function
+               dudc = c -> c^(-gamma),                  # u_prime
+               f = k -> k^alpha,                        # production function
+               f_prime = k -> alpha * k^(alpha - 1))
+    return (; alpha, beta, gamma, mu, s, grid, grid_min, grid_max, grid_size, u,
```

Comment:
Kind of weird spacing.  Is this what it did?  If so, I would put them on one line and then rereun the formatter.  You probably had an old carriage return there before `f_prime`` before